### ISSUE TITLE
"Invalid cookie value" after password change, but OK after refresh!

### ISSUE THREAD

**Issue by wolfcat** [hapijs/yar#65]
After changing the authenticated cookie password, restarting the app results in a {"statusCode":400,"error":"Bad Request","message":"Invalid cookie value"} response from the server. But after a refresh of the page, the proper page displays instead of the 400. I have set clearInvalid to true inside the cookieOptions as shown below:
```javascript
server.register({
	register: require('yar'),
	options: {
		cookieOptions: {
        	password: 'pass',
        	clearInvalid: true,
        	isSecure: false
    	}
	}
}, function (err) {});
```
I am using Hapi version 8.0.0 and yar version 3.0.3

**Comment by ankitpatial** [comment]
+1

**Comment by ankitpatial** [comment]
no use of refresh, needs to clear cache to make it work.

**Comment by wolfcat** [comment]
Clearing the cache also makes it works...

**Comment by gkatsev** [comment]
I'm seeing the same issue. It wouldn't work initially at all but after adding 'clearInvalid', it would work like in the main issue.

**Comment by mark-bradshaw** [comment]
Fixed with #80, and released as v4.0.0.
